cran-comments.md

```diff
@@ -1,20 +1,13 @@
 ## Test environments
 
-release 4.0.4
+release 4.0.3
 
 * OSX (local) - release
 * OSX (actions) - release
-* Ubuntu (actions) - 3.5, 3.6, oldrel, release and devel
+* Ubuntu (actions) - 3.4 to 3.6, oldrel, release and devel
```

Comment:
It only actually tests 3.5 onwards - the cran-comments need updating to reflect this.